labelbox/schema/export_filters.py

```diff
@@ -0,0 +1,12 @@
+import sys
+
+from typing import Optional
+if sys.version_info >= (3, 8):
+    from typing import TypedDict, Tuple
+else:
+    from typing_extensions import TypedDict
+
+
+class ProjectExportFilters(TypedDict):
+    label_created_at: Optional[Tuple[str, str]]
```

Comment:
This looks like a good place to add a docstring to describe each of these filter items in detail, and maybe put the behavior for how we fetch the timezones as well

Reply:
I think we could do a follow up PR with improved docs. At this point we're linking to the docs through URL in export_v2's docstring, which should give good guidance. 

I don't want to block the release with the docs.